client/landing/site.landing/styl/team.styl

```diff
@@ -12,46 +13,67 @@ team.styl
 */
 }
 
+body.team #kdmaincontainer #main-tab-view
+body.teams #kdmaincontainer #main-tab-view
+  height                    auto
+
+// HEADER
 .main-header.team
-  padding                   0 22px
-  max-width                 1200px
-  top                       32px
+  max-width                 100%
+  border-bottom             1px solid rgba(0,0,0,0.15)
+  abs()
+  top                       0
+  height                    69px
+  padding                   0 40px
   z-index                   1
-  fix()
 
-.content-area-pane.team
-.content-area-pane.teams
-  font-family               adelleSans
-  bg                        color #111
+  .koding-header-logo
+    left                    auto
+    top                     18px
 
-  &:before
-    background              -webkit-linear-gradient(274deg,#261a4b 0,#402f7c 48%,#4781c0 100%)
-    background              linear-gradient(176deg,#261a4b 0,#402f7c 48%,#4781c0 100%)
-    background              -webkit-radial-gradient(56% 135%,140% 196%,#50b5e7 0,#402f7c 48%,#000 100%)
-    background              radial-gradient(56% 135%,140% 196%,#50b5e7 0,#402f7c 48%,#000 100%)
+  .hiring-badge
+    top                     21px
 
-    content                 ''
-    fix                     0px 0 0 0px
-    size                    100% 100%
-    z-index                 0
+  nav.full-menu
+    margin-top              21px
+
+
+.content-area-pane.team
+.content-area-pane.teams
```

Comment:
The same explanation as above. Since there is no any child css classes inside of that expression, it doesn't lead to the repetition on the compiled css